ccdaservice/serveccda.js

```diff
@@ -24,8 +24,9 @@ const { fDate, templateDate } = require('./utils/date/date');
 const { countEntities } = require('./utils/count-entities/count-entities');
 const { populateTimezones } = require('./utils/timezones/timezones');
 const {
-    fetchPreviousAddresses,
-} = require('./utils/demographics/previous-addresses');
```

Comment:
was the above line supposed to be removed?

Reply:
Yes, this function is not needed in this file anymore :)